tools/lsp/preview/native.rs

```diff
@@ -262,6 +262,17 @@ pub fn notify_diagnostics(diagnostics: &[slint_interpreter::Diagnostic]) -> Opti
     Some(())
 }
 
+pub fn set_known_components(known_components: Vec<ComponentInformation>) {
```

Comment:
Really this shouldn't be duplicated between native and wasm.
The state is almost the same so it could be shared. 
(can be cleanup in another PR)